modules/api/php/endpoints/candidate/visit/dicoms.class.inc

```diff
@@ -138,20 +156,167 @@ class Dicoms extends Endpoint implements \LORIS\Middleware\ETagCalculator
         case "v0.0.3":
             $view = (new \LORIS\api\Views\Visit\Dicoms(
                 $this->_visit,
-                ...$dicomtars
+                $dicomtars
```

Comment:
maybe my eyes are tricking me but why is there a special case for v0.0.3 when the default code is exactly the same as the v0.0.3?

Reply:
This was a remnant of the original PR, changed.